modules/virt-adding-pci-device-hyperconverged-operator.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-adding-pci-device-hyperconverged-operator_{context}"]
+= Adding the PCI device to the HyperConverged Operator
+
+To enable the availability of a PCI device in the cluster, add details about the PCI device to the HyperConverged Operator in the YAML file. The PCI device is exposed and permitted to be used in the cluster.
+
+.Procedure
+. Use the following command to add the PCI device information to the YAML file.
++
+[source,terminal]
+$ oc get hyperconverged kubevirt-hyperconverged -n openshift-cnv -o yaml
+
++
+.Example
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: openshift-cnv
+spec:
+  permittedHostDevices: <1>
+    pciHostDevices: <2>
+      - pciVendorSelector: 10De:1EB8 <3>
```

Comment:
10DE:1EB8  (Note: Lowercase "e" in "10De") , should not matter , but for consistency sake.

Reply:
+1

Reply:
Updated the content.